azurerm/internal/services/web/resource_arm_app_service_hybrid_connection.go

```diff
@@ -0,0 +1,186 @@
+	relayName := d.Get("relay_name").(string)
+	relayNameArmUri := d.Get("relay_arm_uri").(string)
+	hostname := d.Get("hostname").(string)
+	port := int32(d.Get("port").(int))
+	serviceBusSuffix := d.Get("service_bus_suffix").(string)
+	serviceBusNamespace := d.Get("service_bus_namespace").(string)
+	sendKeyName := d.Get("send_key_name").(string)
+	sendKeyValue := d.Get("send_key_value").(string)
+
+	hybridConnectionProperties := web.HybridConnectionProperties{
+		ServiceBusNamespace: &serviceBusNamespace,
+		RelayName:           &relayName,
+		RelayArmURI:         &relayNameArmUri,
+		Hostname:            &hostname,
+		Port:                &port,
+		SendKeyName:         &sendKeyName,
+		SendKeyValue:        &sendKeyValue,
+		ServiceBusSuffix:    &serviceBusSuffix,
+	}
+
+	connectionEnvelope := web.HybridConnection{}
+	connectionEnvelope.HybridConnectionProperties = &hybridConnectionProperties
+
+	if _, err := client.CreateOrUpdateHybridConnection(ctx, resGroup, name, namespaceName, relayName, connectionEnvelope); err != nil {
+		return fmt.Errorf("Error creating App Service Hybrid Connection %q (resource group %q): %s", name, resGroup, err)
+	}
+	return resourceArmAppServiceHybridConnectionRead(d, meta)
+}
+
+func resourceArmAppServiceHybridConnectionRead(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*clients.Client).Web.AppServicesClient
+	ctx, cancel := timeouts.ForRead(meta.(*clients.Client).StopContext, d)
+	defer cancel()
+
+	id, err := azure.ParseAzureResourceID(d.Id())
+	if err != nil {
+		return err
+	}
+	resGroup := id.ResourceGroup
+	name := id.Path["sites"]
+	namespaceName := id.Path["hybridConnectionNamespaces"]
+	relayName := id.Path["relays"]
+
+	resp, err := client.GetHybridConnection(ctx, resGroup, name, namespaceName, relayName)
+	if err != nil {
+		if utils.ResponseWasNotFound(resp.Response) {
+			d.SetId("")
+			return nil
+		}
+		return fmt.Errorf("Error making Read request on App Service Hybrid Connection %q in Namespace %q, Resource Group %q: %s", name, namespaceName, resGroup, err)
+	}
+	d.Set("app_service_name", name)
+	d.Set("resource_group_name", resGroup)
+	d.Set("namespace_name", namespaceName)
+	d.Set("relay_name", relayName)
+
+	if props := resp.HybridConnectionProperties; props != nil {
+		d.Set("port", resp.Port)
+		d.Set("service_bus_namespace", resp.ServiceBusNamespace)
+		d.Set("send_key_name", resp.SendKeyName)
+		d.Set("send_key_value", resp.SendKeyValue)
+		d.Set("service_bus_suffix", resp.ServiceBusSuffix)
+		d.Set("relay_arm_uri", resp.RelayArmURI)
+		d.Set("hostname", resp.Hostname)
+	}
+
+	return nil
+}
+
+func resourceArmAppServiceHybridConnectionDelete(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*clients.Client).Web.AppServicesClient
+	ctx, cancel := timeouts.ForDelete(meta.(*clients.Client).StopContext, d)
+	defer cancel()
+
+	id, err := azure.ParseAzureResourceID(d.Id())
+	if err != nil {
+		return err
+	}
+	resGroup := id.ResourceGroup
```

Comment:
for consistency could we make this
```suggestion
	resourceGroup := id.ResourceGroup
```